projects/project-2/assignment/reviewers/GDC_lv6.sparql

```diff
@@ -0,0 +1,24 @@
+# checking whether all GDCs are generically depending on something
```

Comment:
It seems to me that your queries are much more complex than mine and the others I have reviewed, we may have to see how to adjust the complexity levels.
I have left comments where protege gives me an error in the execution.
The only thing I would consider is whether to add a short description of the functionality to each one (at least it seemed to me that the assignment required title + description)